src/constants/apps.ts

```diff
@@ -4,4 +4,5 @@ export default {
     APPS_BINDINGS_POST_MENU_ITEM: '/post_menu',
     APPS_BINDINGS_CHANNEL_HEADER_ICON: '/channel_header',
     APPS_BINDINGS_COMMAND: '/command',
+    APPS_BINDINGS_IN_POST: '/in_post',
```

Comment:
Random idea: Are we considering more stuff to add "in_post" apart of embedded forms? I cannot think of any useful idea, but I still see potential.